File: output/CodeLlama/codenet/C++/Python/s097779718.py
def Euclid(x, y):
	tmp = 0

	if x < y:
		tmp = x
		x = y
		y = tmp

	if x % y == 0:
		return y
	return Euclid(y, x % y)

A = int(input())
B = int(input())

print((A*B) / Euclid(A, B))

#include <iostream>
using namespace std;

int Euclid(int x, int y) {
	int tmp;

	if (x < y) {
		tmp = x;
		x = y;
		y = tmp;
	}

	if (!(x%y)) return y;
	return Euclid(y, x%y);
}

int main() {
	int A, B; cin >> A >> B;

	cout << (A*B) / Euclid(A, B) << endl;
}

File: output/CodeLlama/codenet/C++/Java/s097779718.java
import java.util.Scanner;

public class s097779718 {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		long A = sc.nextLong();
		long B = sc.nextLong();
		long tmp;

		if (A < B) {
			tmp = A;
			A = B;
			B = tmp;
		}

		if (A % B == 0) {
			System.out.println(B);
		} else {
			System.out.println(Euclid(A, B));
		}
	}

	public static long Euclid(long x, long y) {
		long tmp;

		if (x < y) {
			tmp = x;
			x = y;
			y = tmp;
		}

		if (x % y == 0) {
			return y;
		} else {
			return Euclid(y, x % y);
		}
	}
}